tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -7,9 +7,7 @@
 
 
 class ElementOptions(Dict[str, Any]):
-    def __init__(
-        self, element: Mapping[str, Any]
-    ):
+    def __init__(self, element: Mapping[str, Any]):
```

Comment:
Looks like we missed the call to `dict.__init__()` here as well, but up to you to fix it now or later.

Reply:
@jrandolf please follow up on this once you are back